packages/parser/tests/test-utils.ts

```diff
@@ -0,0 +1,13 @@
+import type { LangiumParser } from 'langium';
+
+import { Info, InfoServices, createInfoServices } from '../src/language/index.js';
+
+const infoServices: InfoServices = createInfoServices().Info;
+const infoParser: LangiumParser = infoServices.parser.LangiumParser;
+export function createInfoTestServices() {
+  const parse = (input: string) => {
+    return infoParser.parse<Info>(input);
+  };
+
+  return { services: infoServices, parse };
+}
```

Comment:
Shouldn't this be in info.test.ts as there's nothing common here?

Reply:
It was taken from the common function `createMermaidServices`, so I placed in common place.